src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
+#include <boost/algorithm/string.hpp>
+#include <sstream>
 
 namespace ripple {
 
+class Definitions
+{
+private:
+    std::string
+    translate(std::string inp)
+    {
+        auto replace = [&](const char* f, const char* r) -> std::string {
```

Comment:
f and r seem like strange param names. Maybe give them more descriptive names?

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/d558b665cd2658b8fd6d2176662758f9efd8ae3f